UIKitDZMain/ViewController.swift

```diff
@@ -8,9 +8,22 @@
 import UIKit
 
 final class ViewController: UIViewController {
-
+    
+    var hidePass = false
```

Comment:
```suggestion
    var isHidePass = false
```
ниже аутлета перенеси как оп сайду из важного